packages/apollo-server-core/src/runHttpQuery.ts

```diff
@@ -288,6 +293,13 @@ export async function runHttpQuery(
 
   const responses = await Promise.all(requests);
 
+  const responseInit: ResponseInit = {
+    status: 200,
```

Comment:
Seems like this isn't getting consistently used in all variants (even express lacks it).